src/modules/setup/setupTSConfig.ts

```diff
@@ -24,12 +24,15 @@ const getTSConfig = async () => {
   }
 }
 
-export const setupTSConfig = async (manifest: Manifest) => {
+export const setupTSConfig = async (manifest: Manifest, warnOnNoBuilderCandidate: boolean) => {
   log.info(`Setting up tsconfig.json`)
   try {
     const tsconfigsFromBuilder = await getTSConfig()
-    if (!tsconfigsFromBuilder) {
-      log.warn(`No builders candidates for TSConfig setup`)
+    if (!tsconfigsFromBuilder ) {
+      if(warnOnNoBuilderCandidate) {
+        log.warn(`No builders candidates for TSConfig setup`)
+      }
```

Comment:
```suggestion
    if (!tsconfigsFromBuilder && warnOnNoBuilderCandidate) {
      log.warn(`No builders candidates for TSConfig setup`)
```

Reply:
Formatting github-action got it! Just fixed haha